Handle mobileconfig display names with special characters

emericklaw opened the pull request:
If the account display name contained special characters like & the mobileconfig would fail to import on the iOS device.

emericklaw committed:
Handle mobileconfig display names with special characters

If the account display name contained special characters like & the mobileconfig would fail to import on the iOS device.

mkuron commented:
You'll only want to replace ampersands and less/greater signs. Your current pull request also replaces double quotes (which carry no special meaning inside XML tags), so you should add `ENT_NOQUOTES`.

emericklaw committed:
Updated to not convert quotes

I missed using ENT_NOQUOTES since XML only needs & and <> to be replaced in tags, spotted by @mkuron

emericklaw commented:
Thanks @mkuron I have updated
